public/components/custom_panels/panel_modules/visualization_container/visualization_container.scss

```diff
@@ -58,3 +58,28 @@
 .panels-title-text {
   @include euiTextTruncate;
 }
+
+.osd-viz-panel {
+  .euiPanel {
+    box-shadow: none !important;
```

Comment:
is there a better way to not use !important?

Reply:
Need to check on this. Usually, we create a new class for selectivity. That was not possible in this case as my class was on same level with EUI.